modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3421,6 +3421,91 @@ definition {
 			actionLabel = "Update Action Label");
 	}
 
+	@description = "LPS-173537 - Verify adding a Commerce Product entry triggers an action to update the Commerce Product Group entry."
+	@priority = 5
+	test CanUpdateCommerceProductGroupEntryAfterCreatingCommerceProductEntry {
+		property portal.acceptance = "true";
+
+		task ("Given: A Commerce Product Group") {
+			CommerceNavigator.gotoPortlet(
+				category = "Pricing",
+				portlet = "Product Groups");
+
+			Click(locator1 = "Button#PLUS");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "IFrame#MODAL_IFRAME");
+
+			Type(
+				key_fieldLabel = "Name",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "PG1");
+
+			Button.clickSubmitButton();
+		}
+
+		task ("And given: An action to add a Commerce Product Group after updating a Commerce Product Group entry in the Commerce Product Group object") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Product Group");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				actionName = "CustomAction",
+				active = "true",
+				inputMethod = "true",
+				inputValue = "PG2",
+				objectLabel = "Commerce Product Group",
+				thenAction = "Add an Object Entry",
+				whenAction = "On After Update");
+		}
+
+		task ("When: The user updates an Commerce Product Group entry") {
+			CommerceNavigator.gotoPortlet(
+				category = "Pricing",
+				portlet = "Product Groups");
+
+			CommerceEntry.clickEntryDropdownMenuItem(
+				entryName = "PG1",
+				menuItem = "Edit");
+
+			Type(
+				key_fieldLabel = "Name",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "PG1 Edited");
+
+			Button.clickSave();
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Then: The action is triggered and the Commerce Product Group entry is added") {
```

Comment:
Rename to: `task ("Then: Then the action is triggered and the Commerce Product Group entry is updated")`

Reply:
We are following this pattern to tasks (without the use colon)